onnx/onnx.in.proto

```diff
@@ -497,6 +497,50 @@ message TensorShapeProto {
   repeated Dimension dim = 1;
 }
 
+// Sequences
+//
+// Defines a dense, ordered, collection of elements that are of homogeneous types.
+message SequenceProto {
+  // Sequences are made of TypeProto elements, which can either be tensors,
+  // maps, or sequences themselves.
+  message SequenceElement {
```

Comment:
I think this and MapElement below should be the same (shared) message-type. (We could probably copy the style used in the definition of TypeProto, in case the mutual recursive dependence causes a problem.) I think we should allow a sparse-tensor here too.

Reply:
Created shared SequenceMapElement and added sparse tensor support.